vuepress/docs/refarch/README.md

```diff
@@ -1,14 +1,362 @@
-# DDD-Outline
-Here is a general outline of the BCs in use for Mojaloop 2.0.
+<p style="color: red; font-weight: bold">>>>>>  gd2md-html alert:  ERRORs: 0; WARNINGs: 0; ALERTS: 5.</p>
+<ul style="color: red; font-weight: bold"><li>See top comment block for details on ERRORs and WARNINGs. <li>In the converted Markdown or HTML, search for inline alerts that start with >>>>>  gd2md-html alert:  for specific instances that need correction.</ul>
 
-![BC-Overview](./assets/images/BC-overview.png)
+<p style="color: red; font-weight: bold">Links to alert messages:</p><a href="#gdcalert1">alert1</a>
```

Comment:
We can remove this. This looks like a warning from the converter.

Same with these "alert" lines.